content/dev_meetings/kokkos_kernels.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Kokkos Kernels devs meeting"
+date: "2025-08-28"
+---
+
+Overview
+--------
+The Kokkos Kernels developers meeting is intended to give a time to the developers to meet and discuss the needs of the library.
+The discussion typically focuses on new features development, algorithms improvements, bug fixes, platforms support, interaction and support for vendors libraries and community events organization.
+Occasionally new reseach topics in linear algebra, linear solvers and adjacent topics are scheduled.
+
+Connect
+-------
+* **Slack channel:** [# kokkos-kernels](https://kokkosteam.slack.com/archives/CMZDYKZ96)
```

Comment:
Is that a Kokkos Kernels support channel or also for developers

Reply:
That's the general Kokkos Kernels channel for support so it should be appropriate for new people to ask initial questions about the meeting there.